lib/help_scout/api/client.rb

```diff
@@ -13,7 +13,7 @@ def connection
         @_connection ||= build_connection.tap do |conn|
           if authorize?
             HelpScout::API::AccessToken.refresh!
-            conn.authorization(:Bearer, access_token) if access_token
+            conn.request :authorization, 'Bearer', 'authentication-token' if access_token
```

Comment:
@mfittko I think this isn't working yet, because when I run a HelpsScout query in the console of the  [update-bugsnag-faraday](https://github.com/sofatutor/sofatutor/pull/12611) branch, I get the error `HelpScout::API::NotAuthorized: The access token is invalid or has expired` I don't get the error if I try that on master. Also the same error shows up on bugsnag for the testing_cobain_testing stage [on bugsnag](https://app.bugsnag.com/sofatutor-gmbh/sofatutor/errors/65dd0a2aaa29cd0008c82918?filters[event.since]=30d&filters[error.status]=open&filters[search]=helpscout&pivot_tab=event) from last nights cron-jobs.

Reply:
ouch, yeah we need to use the token not some random string 🤦